cloud/jenkins/pgo_operator_eks_latest.groovy

```diff
@@ -218,6 +213,7 @@ EOF
             export KUBECONFIG=/tmp/${CLUSTER_NAME}-${CLUSTER_SUFFIX}
             export PATH=/home/ec2-user/.local/bin:$PATH
             eksctl create cluster -f cluster-${CLUSTER_SUFFIX}.yaml
+            kubectl annotate storageclass gp2 storageclass.kubernetes.io/is-default-class=true
```

Comment:
we need to add this annotation for all EKS jenkins jobs

Reply:
I will, but in another PR. I need to fix this pipeline first, so that I can complete the post-release steps for PG operator, which include creating a release tag in this repo `pg-operator-2.4.0`.